src/tqec/utils/noise_model.py

```diff
@@ -519,14 +522,11 @@ def _iter_split_op_moments(
                 yield cur_moment
                 cur_moment = []
             yield op
-        else:  # if isinstance(op, stim.CircuitInstruction):
-            if op.name == "TICK":
-                yield cur_moment
-                cur_moment = []
-            else:
-                cur_moment.extend(
-                    _split_targets_if_needed(op, immune_qubits=immune_qubits)
-                )
+        elif op.name == "TICK":
+            yield cur_moment
+            cur_moment = []
+        else:
+            cur_moment.extend(_split_targets_if_needed(op, immune_qubits=immune_qubits))
```

Comment:
To do: Check the logic of this change made by ruff.

Reply:
That seems correct.

Reply:
Thanks, I made this comment late last night for today. :)